corehq/apps/analytics/static/analytix/js/hubspot.js

```diff
@@ -44,28 +44,15 @@ hqDefine('analytix/js/hubspot', [
      * Activates the Hubspot Request Demo form
      */
     _utils.loadDemoForm = function () {
-        let isTrial = _get('isDemoTrial'),
-            isVariant = _get('demoABv2') && _get('demoABv2').version === 'variant',
```

Comment:
Can you delete the A/B test altogether? That would include the test definition [here](https://github.com/dimagi/commcare-hq/blob/e2b4bd82d4cc4076823df78cc49ec357ac7b7c7b/corehq/apps/analytics/ab_tests.py#L140-L147), and then places that reference the test, like initial page data [here](https://github.com/dimagi/commcare-hq/blob/e2b4bd82d4cc4076823df78cc49ec357ac7b7c7b/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L263).

Reply:
I did not feel comfortable doing this. When I investigated and saw that the test also affects Kissmetrics [here](https://github.com/dimagi/commcare-hq/blob/3ed301fcbe38af2719fc0dfa9fc069f968992e88/corehq/apps/hqwebapp/templates/hqwebapp/base.html#L261-L264), I felt this was outside the scope of this change and required context of what the test was meant to do.